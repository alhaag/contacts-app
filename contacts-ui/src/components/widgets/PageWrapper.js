import React from 'react'

const PageWrapper = (props) => {
    return (
        <div style={styles.wrapper}>
            {props.children}
        </div>
    )
}

const styles = {
    wrapper: {
        padding: '20px',
    }
}

export default PageWrapper