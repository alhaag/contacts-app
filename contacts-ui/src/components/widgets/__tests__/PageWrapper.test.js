import React from 'react'
import renderer from 'react-test-renderer'
import PageWrapper from '../PageWrapper';

test('Presentation PageWrapper component', () => {

  const component = renderer.create(
    <PageWrapper>
      <p>Testing content</p>
    </PageWrapper>,
  );
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()

})