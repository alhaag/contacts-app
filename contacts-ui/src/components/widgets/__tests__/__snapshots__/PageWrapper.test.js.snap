// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Presentation PageWrapper component 1`] = `
<div
  style={
    Object {
      "padding": "20px",
    }
  }
>
  <p>
    Testing content
  </p>
</div>
`;
